pkg/reconciler/events/cloudevent/cloud_event_controller.go

```diff
@@ -40,24 +38,6 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-// InitializeCloudEvents initializes the CloudEvents part of the
-// TaskRunStatus from a slice of PipelineResources
-func InitializeCloudEvents(tr *v1beta1.TaskRun, prs map[string]*resource.PipelineResource) {
-	// If there are no cloud event resources, this check will run on every reconcile
-	if len(tr.Status.CloudEvents) == 0 {
-		var targets []string
-		for name, output := range prs {
-			if output.Spec.Type == resource.PipelineResourceTypeCloudEvent {
-				cer, _ := cloudevent.NewResource(name, output)
-				targets = append(targets, cer.TargetURI)
-			}
-		}
-		if len(targets) > 0 {
-			tr.Status.CloudEvents = cloudEventDeliveryFromTargets(targets)
-		}
-	}
-}
-
 func cloudEventDeliveryFromTargets(targets []string) []v1beta1.CloudEventDelivery {
```

Comment:
Are any other functions in this file now dead code?

Reply:
Thanks for the reminder, looking into it!

Reply:
Done, not any dead code that could be removed with CloudEvent resources.

Reply:
There are two code paths for CloudEvents, one is associated with the pipeline resource and the other one with the CloudEvents feature.  If I remember correctly, `SendCloudEvents` is used by the former and `SendCloudEventsWithRetries` is used by the latter. 
I need to spend a bit more time reviewing the code to remember exactly what can be removed. I would recommend to try to remove the  `SendCloudEvents` and related usages and check if all cloud events test cases are still passing.

Reply:
Thanks Andrea, deleted:
- cloudEventDeliveryFromTargets
- SendCloudEvents